### DISCUSSION TITLE
easywechat找不到codeToSession

### DISCUSSION THREAD

**Discussion by ElQut** [w7corp/easywechat#2495] [General]
easywechat找不到codeToSession
`$app = new Application($config);
                $utils = $app->getUtils();
$utils->codeToSession($option["code"]);`
![WX20220608-144915](https://user-images.githubusercontent.com/35054752/172550607-d44f4c69-efcf-4732-98eb-cbdcb57615e3.png)

**Comment by overtrue** [comment]
我一猜就是用了已经没有维护的阿里云的 packagist 源。。。删掉，更新到最新版。

**Comment by jncinet** [comment]
阿里云服务器上，用不了腾讯云的镜像

**Reply by overtrue** [reply]
这不是我的问题，要么去找阿里云解决，要么用官方源啊……
